deploy/push.sh

```diff
@@ -366,49 +396,50 @@ function push_phylesystem_api {
     [ "x$AMENDMENTS_REPO"   != "x" ] || err "AMENDMENTS_REPO not configured"
     [ "x$FAVORITES_REPO"    != "x" ] || err "FAVORITES_REPO not configured"
     [ "x$OTI_BASE_URL"      != "x" ] || err "OTI_BASE_URL not configured"
+    [ "x$READ_ONLY_MODE"    != "x" ] || err "READ_ONLY_MODE not configured"
     [ "x$OTINDEX_BASE_URL"      != "x" ] || err "OTINDEX_BASE_URL not configured"
 
-    push_bot_identity
+    push_bot_identity || exit 1
 
     # Place private key for GitHub access
     if [ "x$OPENTREE_GH_IDENTITY" = "x" ]; then
-        echo "Warning: OPENTREE_GH_IDENTITY not specified"
+        echo "Warning: OPENTREE_GH_IDENTITY not specified" || exit 1
     elif [ ! -r $OPENTREE_GH_IDENTITY ]; then
-        echo "Warning: $OPENTREE_GH_IDENTITY not found"
+        echo "Warning: $OPENTREE_GH_IDENTITY not found" || exit 1
     else
-        rsync -p -e "${SSH}" "$OPENTREE_GH_IDENTITY" "$OT_USER@$OPENTREE_HOST":.ssh/opentree
-        ${SSH} "$OT_USER@$OPENTREE_HOST" chmod 600 .ssh/opentree
+        rsync -p -e "${SSH}" "$OPENTREE_GH_IDENTITY" "$OT_USER@$OPENTREE_HOST":.ssh/opentree || exit 1
+        ${SSH} "$OT_USER@$OPENTREE_HOST" chmod 600 .ssh/opentree || exit 1
     fi
 
     # Try to place an OAuth token for GitHub API by bot user 'opentreeapi'
     tokenfile=${OPENTREE_SECRETS}/OPENTREEAPI_OAUTH_TOKEN
```

Comment:
I'm a bit confused about what should be happening here. I should have the OPENTREE_OAUTH_TOKEN saved in .ssh/opentree/ on my local machine, and tehn it gets copied over? Is this how it should still work?

Reply:
Ah - following conversation with Mark I believe we are using a keypair generated directly on phylsystemapi/devphylesystemapi for the pushes to phylesystem. So I *think* we don't need this bit.

Reply:
Hmm it looks like it also tries to grab the opentree key here https://github.com/OpenTreeOfLife/germinator/pull/155/files#diff-028ee22cd145bc1227616bfad2602bdeR410